pages/linux/rename.ul.md

```diff
@@ -0,0 +1,29 @@
+# rename.ul
+
+> Renames multiple files.
+> This is the util-linux version of `rename`. See `rename` for the perl version which is default on Debian.
+> Warning: The renaming has no safeguards and will overwrite files without any questions.
```

Comment:
Perhaps this will read better?
```suggestion
> Warning: This command has no safeguards and will overwrite files without prompting!
```

Reply:
Much better, thanks!

Reply:
@sbrl  the exclamation mark broke it again, I forgot. :/ Do I need to change it or does the test need to be changed?